src/Searchlight/Parsing/SyntaxParser.cs

```diff
@@ -410,6 +410,17 @@ private static BaseClause ParseOneClause(SyntaxTree syntax, DataSource source, T
             {
                 return null;
             }
+            
+            if (columnInfo.IsEncrypted && (op != OperationType.Equals && op != OperationType.IsNull && op != OperationType.In))
```

Comment:
feedback: maybe simplify this to a list of allowed ops, so you can do `!EncryptedOperations.Contains(op)`

Reply:
updated